### PR TITLE
FEATURE: Add chat and sidebar toggles to the setup wizard

### PR DESCRIPTION
**Pull request by oblakeerickson**
- Fix css alignment
- Add Enable Chat Toggle
- Add Enable Sidebar Toggle

image

image

### CODE REVIEW

File: lib/wizard/step_updater.rb [hunk 1]
```diff
@@ -37,6 +37,10 @@ def update_setting(id, value)
         value = Upload.get_from_url(value) || ''
       end
 
+      if id == :navigation_menu
+        value = value.to_s == "true" ? NavigationMenuSiteSetting::SIDEBAR : NavigationMenuSiteSetting::LEGACY
```
Comment (tgxworld) [inline lib/wizard/step_updater.rb:41]:
```suggestion
        value = value.to_s == "true" ? NavigationMenuSiteSetting::SIDEBAR : NavigationMenuSiteSetting::HEADER_DROPDOWN
```

@mcwumbly Is the goal to still drop back to legacy here?

Comment (mcwumbly) [reply lib/wizard/step_updater.rb:41]: @tgxworld no, I think you have it right. The toggle here should be choosing between `SIDEBAR` and `HEADER_DROPDOWN`.  We can reserve `LEGACY` for existing sites (new sites can go out of their way to choose it).

Comment (mcwumbly) [reply lib/wizard/step_updater.rb]: @oblakeerickson looks like a test also needs updating?

```
  1) Wizard::StepUpdater privacy updates to open correctly
     Failure/Error: expect(SiteSetting.navigation_menu).to eq(NavigationMenuSiteSetting::LEGACY)

       expected: "legacy"
            got: "header dropdown"
     ```

**Review by pmusaraj — APPROVED** [review]
Code looks good, would love to see a couple of screenshots, though!

File: app/assets/stylesheets/wizard.scss [hunk 2]
```diff
@@ -618,15 +618,27 @@ body.wizard {
   &__checkbox-slider:before,
   &__checkbox-slider:after {
     content: "";
+    display: block;
+    position: absolute;
+  }
+
+  &__checkbox-slider:after {
+    content: "\2713"; // checkmark
```
[the remark this answers is not in the document]
Comment (pmusaraj) [reply app/assets/stylesheets/wizard.scss:626]: Thanks!

**Comment by tgxworld** [comment]
@oblakeerickson Just a note for the future, do take note of cleaning up the commit message when squashing and merging on Github in the future. Since we're squashing the commits, the commit message should not need to include all the commit messages of commits made during the review.

**Pull request merged**